### PR TITLE
add nimbus vc

### PR DESCRIPTION
**Pull request by xenowits**
Adds two nimbus VCs to `docker-compose.yml` by replacing one teku VC and one lighthouse VC.

Ticket: https://github.com/ObolNetwork/charon-distributed-validator-cluster/issues/67

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -192,18 +192,18 @@ services:
...
-      ETH2_NETWORK: ${ETH2_NETWORK:-goerli}
+    command: |
+      --beacon-node=http://node2:3600
+      --secrets-dir=/home/user/validator_keys
+      --validators-dir=/home/user/validator_keys
     volumes:
-      - ./lighthouse/run.sh:/opt/lighthouse/run.sh
-      - .charon/cluster/node2/validator_keys:/opt/charon/keys
+      - .charon/cluster/node2/validator_keys:/home/user/validator_keys
+      - ./nimbus:/home/user/data
```
Comment (xenowits) [inline docker-compose.yml:206]: Mounting to `/home/user/data` as recommended by docs - https://nimbus.guide/docker.html
Screenshot 2023-02-06 at 23 15 53

File: nimbus/sample.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+# This is a sample file so that the `nimbus/` folder is pushed to git.
```
Comment (xenowits) [inline nimbus/sample.txt:1]: Don't think nimbus VC needs a directory so we can probably delete it

File: nimbus/run.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  echo "$password" | \
+  /home/user/nimbus_beacon_node deposits import \
+  --data-dir=/home/user/data/${NODE} \
+  /home/validator_keys
+done
+
+echo "Imported keys"
+
+# Now run nimbus VC
+exec /home/user/nimbus_validator_client --data-dir=/home/user/data/${NODE} --beacon-node=http://192.168.1.10:5052
```
Comment (dB2510) [inline nimbus/run.sh:19]: beacon node endpoint should be charon's endpoint like http://node0:3600

File: docker-compose.yml [hunk 4]
```diff
@@ -231,18 +230,17 @@ services:
...
+    build: nimbus
     networks: [ cluster ]
     depends_on: [ node5 ]
     restart: unless-stopped
-    command: |
-      validator-client
-      --beacon-node-api-endpoint="http://node5:3600"
-      --config-file "/opt/charon/teku/teku-config.yaml"
+    environment:
+      BEACON_NODE_ADDRESS: http://node2:3600
```
Comment (dB2510) [inline docker-compose.yml:239]:
```suggestion
      BEACON_NODE_ADDRESS: http://node5:3600
```

**Review by dB2510 — APPROVED** [review]

**Pull request merged**